pkg/acs/lib/acsccd/doccd.c

```diff
@@ -437,37 +364,15 @@ int DoCCD (ACSInfo *acs_info) {
 
                  if (doBlev(&acs[i], &x[i], acs[i].chip, &meanblev, &done, &driftcorr)) {
                     freeOnExit (&ptrReg);
-                    return (status);
+                    return status;
                  }
 
                  /* Variable done is overwritten in doBlev if bias is determined from overscan region */
-                 if (done) {
-                    trlmessage("Bias level from overscan has been subtracted;");
-
-                    if (PutKeyFlt(&x[i].sci.hdr, "MEANBLEV", meanblev,
-                                   "mean of bias levels subtracted in electrons")) {
-                       freeOnExit (&ptrReg);
-                       return (status);
-                    }
-
-                    trlmessage("     mean of bias levels subtracted was %.6g electrons.", meanblev);
-
-                 } else {
-                       trlmessage("Default bias level from CCDTAB was subtracted.");
+                 if (!done) {
+                     trlmessage("Default bias level from CCDTAB was subtracted.");
```

Comment:
I am a little fuzzy here.  Why did the **_if_** statement change?

Reply:
I moved the MEANBLEV calculation to `doblev.c`. I don't see why it has to be repeated 3x in `doccd.c`. However, the "Default bias level from CCDTAB was subtracted." message logic applies outside of `doblev.c` and there is a comment that `done` controls whether the value is from CCDTAB or not. Does this explain my changes?